stdlib/process.jou

```diff
@@ -9,6 +9,12 @@ declare system(command: byte*) -> int
 # Exit the current process.
 declare exit(status: int) -> noreturn  # Clean exit. Status 0 is success, other status (usually 1) means fail.
 declare abort() -> noreturn  # Hard exit. Similar to a crash.
+declare execl(path: byte*, arg: byte*, ...) -> int # Execute the executable file with argument list args, replacing the current process.
```

Comment:
This function doesn't exist on Windows. We can add it later when it's possible to declare a function only on some platforms.